### PR TITLE
Feature 1 - wdrożenie

### PR DESCRIPTION
**Pull request by jbossops**
Feature 1 - wdrożenie

### CODE REVIEW

File: file1.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+file1.txt
```
Comment (LinusTorval) [inline file1.txt:1]: Change OK

Comment (LinusTorval) [reply file1.txt:1]: Even better

File: file2.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+file2.txt
```
Comment (LinusTorval) [inline file2.txt:1]: Change OK

Comment (LinusTorval) [reply file2.txt:1]: Even better

Comment (jbossops) [reply file2.txt:1]: OK

**Review by jbossops — COMMENTED** [review]
OK all chnages are ok

**Review by LinusTorval — APPROVED** [review]
All right

**Pull request merged**